src/praisonai-agents/praisonaiagents/llm/llm.py

```
@@ -1893,26 +1896,47 @@ def _prepare_return_value(text: str) -> Union[str, tuple]:
+                                    tool_call_id=tool_call_id,
+                                    is_ollama=is_ollama
+                                ))
+                            
+                            # Create appropriate executor based on parallel_tool_calls setting
+                            executor = create_tool_call_executor(parallel=parallel_tool_calls)
+                            
+                            # Execute batch
+                            tool_results_batch = executor.execute_batch(tool_calls_batch, execute_tool_fn)
+                            
+                            tool_results = []
+                            for tool_call_obj, tool_result_obj in zip(tool_calls_batch, tool_results_batch):
```

Ruff (0.15.10)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)